packages/components/psammead-image/src/index.jsx

```diff
@@ -6,7 +6,7 @@ export { default as AmpImg } from './index.amp';
 
 const StyledImg = styled.img`
   display: block;
-  width: 100%;
```

Comment:
Is there a reason we wouldn't want this to have both a `width` and a `max-width`? 
The idea being the `img` should always be 100% width unless it reaches the max-width